.github/workflows/social-rfc-update.yml

```diff
@@ -11,18 +11,17 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: '18.x'
-      - run: npx @humanwhocodes/tweet "The RFC '$TITLE' is now in the ${{ github.event.label.name }} phase.\n\n${{ github.event.pull_request.html_url }}"
+      - run: npx @humanwhocodes/crosspost -t -b -m "The RFC '$TITLE' is now in the ${{ github.event.label.name }} phase.\n\n${{ github.event.pull_request.html_url }}"
         env:
-          TITLE: ${{ github.event.pull_request.title }}
```

Comment:
I believe that `TITLE` is still needed because it's used in the crosspost message (`"The RFC '$TITLE' `).

Reply:
Oops, copy-paste error. Thanks!